compoments/report-repair/priority-list.js

```diff
@@ -44,4 +43,4 @@ PriorityList.propTypes = {
   handleChange: PropTypes.func,
 }
 
-export default PriorityList;
+export default withRouter(PriorityList)
```

Comment:
This export is `withRouter`, but doesn't use a `router` - is it required?

This is the only component exporting using withRouter too.
I've looked at the [documentation](https://nextjs.org/docs/api-reference/next/router) and can't figure out why it's needed in this component, i.e. it doesn't access router by props.
Please help me understand the need for it.